README.md

```diff
@@ -4,27 +4,26 @@
 ![MIT](https://img.shields.io/badge/license-MIT-blue.svg)
 [![docs.rs](https://img.shields.io/docsrs/noisy_bevy)](https://docs.rs/noisy_bevy)
 
-Simple stupid noise primitives for glam types (`Vec2`, `Vec3`) and wgsl.
+Simple stupid noise primitives for glam (`Vec2`, `Vec3`) and WGSL.
 
-Main motivations are:
+- Integrates with Bevy seamlessly
+- Same results on the CPU and GPU (not bit-level perfect, though)
 
-- ergonomic usage with Bevy
-- same results on rust and wgsl (not bit-level perfect, though)
+## Features
 
-![screenshot of an asteroid generated on the gpu and cpu](https://s3.johanhelsing.studio/dump/noisy_asteroid.png)
-
-## Implemented noise primitives:
+- `simplex_noise_2d`
+- `simplex_noise_2d_seeded`
+- `simplex_noise_3d`
+- `fbm_simplex_2d`
+- `fbm_simplex_2d_seeded`
+- `fbm_simplex_3d`
+- `worley_2d`
```

Comment:
I think I prefer keeping the links `[]` here. As that makes the rust-doc links clickable

Reply:
Neat, I didn't know it could do that!